.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Is that correct? (does this mean PRs that are intended to be merged into `master`?)

Reply:
I think so. [The docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags) are not super clear, but that seems to be what's happening (based on the fact that the action *is* executing on this PR). Or rather, strictly speaking, pull requests against the master branch.